src/Symfony/Component/Yaml/Yaml.php

```diff
@@ -21,10 +21,7 @@
 class Yaml
 {
     /**
-     * Parses YAML into a PHP array.
-     *
-     * The parse method, when supplied with a YAML stream (string or file),
-     * will do its best to convert YAML in a file into a PHP array.
+     * Parses a YAML string into a PHP array, a scalar or an stdClass object.
```

Comment:
2.3 doesn't support stdClass. This whole PR started with an intent to remove "file" from the `(string or file)` - 3.0 no longer supports it.